src/TLMessage/TLMessage/ClientMessages/Api/get_common_chats.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TelegramOSINT\TLMessage\TLMessage\ClientMessages\Api;
+
+use TelegramOSINT\TLMessage\TLMessage\ClientMessages\Shared\input_user;
+use TelegramOSINT\TLMessage\TLMessage\Packer;
+use TelegramOSINT\TLMessage\TLMessage\TLClientMessage;
+
+/**
+ * @see https://core.telegram.org/method/messages.getCommonChats
+ */
+class get_common_chats implements TLClientMessage
+{
+    const CONSTRUCTOR = 218777796; // 0xd0a48c4
+
+    /** @var int */
+    private $user_id;
+    /** @var int */
+    private $max_id;
+    /** @var int */
+    private $limit;
+    /** @var int */
+    private $accessHash;
+
+    public function __construct(int $user_id, int $limit, int $max_id = 0, int $accessHash = 0)
```

Comment:
Why accessHash not mandatory?
Would it be possible to build correct `input_user` without accessHash?